std/math.d

```diff
@@ -7082,15 +7120,21 @@ float nextDown(float x) @safe pure nothrow @nogc
 
 @safe pure nothrow @nogc unittest
 {
-    static if (floatTraits!(real).realFormat == RealFormat.ieeeExtended)
+    static if (floatTraits!(real).realFormat == RealFormat.ieeeExtended ||
+               floatTraits!(real).realFormat == RealFormat.ieeeDouble ||
+               floatTraits!(real).realFormat == RealFormat.ieeeExtended53 ||
+               floatTraits!(real).realFormat == RealFormat.ieeeQuadruple)
```

Comment:
Let's see if we can enable the tests for all formats except `RealFormat.ibmExtended`.

Reply:
@n8sh do you need a freebsd/x86 or aarch64 machine to test?

Reply:
@ibuclaw that would be useful

Reply:
Both pass ctfe tests without issue.